pkg/cmd/pulumi/logs.go

```diff
@@ -59,8 +53,8 @@ func newLogsCmd() *cobra.Command {
 			"provider. For example, for AWS resources, the `pulumi logs` command will query\n" +
 			"CloudWatch Logs for log data relevant to resources in a stack.\n",
 		Args: cmdutil.NoArgs,
-		Run: cmd.RunCmdFunc(func(cmd *cobra.Command, args []string) error {
-			ctx := cmd.Context()
+		Run: cmd.RunCmdFunc(func(cobraCmd *cobra.Command, args []string) error {
```

Comment:
Wonder if we should just do a pass turning all these to `command` given the type is `*cobra.Command`